cli/azd/cmd/provision.go

```diff
@@ -125,6 +129,23 @@ func (p *provisionAction) Run(ctx context.Context) (*actions.ActionResult, error
 		defaultTitleNote = "This is a preview. No changes will be applied to your Azure resources."
 	}
 
+	// Get Subscription to Display in Command Title Note
+	subscription, subErr := p.subService.GetSubscription(ctx, p.env.GetSubscriptionId(), p.env.GetTenantId())
```

Comment:
What do we do if subErr isn't nil?

Reply:
@jongio let me know your input on this.

Reply:
@vhvb1989 - Thoughts?

Reply:
My preference is not to stop azd-process on this.  We can just continue.
For similar cases, we just log the error.  See this as an example: https://github.com/Azure/azure-dev/blob/d2eb19673b7eaa4b124eb447ee3ba01218cfb6b8/cli/azd/pkg/account/manager.go#L85